### DISCUSSION TITLE
Anyone help me with notion database integration

### DISCUSSION THREAD

**Discussion by dv64bit** [FlowiseAI/Flowise#796] [Q&A]
I am trying to load document from the notion database, but it is showing the same error message again and again
![image](https://github.com/FlowiseAI/Flowise/assets/72154120/6d649d71-adcf-4372-b35a-d5fedbe2bbb7)
I did check my page id and database id still facing the same issue

**Comment by chungyau97** [comment]
Hi @dv64bit,

Have you connect your [created internal integration](https://www.notion.so/help/create-integrations-with-the-notion-api#create-an-internal-integration) into the page?

Steps:
1. Click **three dots**
2. Under **Connections** section, select your **internal integration**.
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/9b57b966-d27d-4c74-b98d-9ea5d290c4a9)

**Reply by dv64bit** [reply]
Yess I already did that

**Reply by chungyau97** [reply]
Are you using the blue highlighted id?
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/714059c0-5d1e-4205-b0bf-36d2f92773c1)
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/4df663c7-8bcf-417a-87f9-d67a7cf2bcce)
